ERROR: 2022/04/17 20:55:35 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 21:03:32 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 21:09:45 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
INFO: 2022/04/17 21:35:42 [Created the table{
  TableDescription: {
    AttributeDefinitions: [{
        AttributeName: "id",
        AttributeType: "S"
      }],
    CreationDateTime: 2022-04-17 16:06:50.205 +0000 UTC,
    ItemCount: 0,
    KeySchema: [{
        AttributeName: "id",
        KeyType: "HASH"
      }],
    ProvisionedThroughput: {
      NumberOfDecreasesToday: 0,
      ReadCapacityUnits: 10,
      WriteCapacityUnits: 10
    },
    TableArn: "arn:aws:dynamodb:us-east-1:937145466380:table/ShippingAddress",
    TableId: "4d60586e-f607-476b-9587-9d916712f777",
    TableName: "ShippingAddress",
    TableSizeBytes: 0,
    TableStatus: "CREATING"
  }
}]
ERROR: 2022/04/17 21:37:00 [0xc00009f608]
ERROR: 2022/04/17 21:37:29 [0xc000484d20]
ERROR: 2022/04/17 21:37:47 [0xc0000042b8]
ERROR: 2022/04/17 21:51:58 [0xc0004847c8]
INFO: 2022/04/17 21:52:37 [Created the table{
  TableDescription: {
    AttributeDefinitions: [{
        AttributeName: "id",
        AttributeType: "S"
      }],
    CreationDateTime: 2022-04-17 16:23:45.676 +0000 UTC,
    ItemCount: 0,
    KeySchema: [{
        AttributeName: "id",
        KeyType: "HASH"
      }],
    ProvisionedThroughput: {
      NumberOfDecreasesToday: 0,
      ReadCapacityUnits: 10,
      WriteCapacityUnits: 10
    },
    TableArn: "arn:aws:dynamodb:us-east-1:937145466380:table/ShippingAddress",
    TableId: "05733701-a02e-4e41-a0fb-d0af95f85cff",
    TableName: "ShippingAddress",
    TableSizeBytes: 0,
    TableStatus: "CREATING"
  }
}]
ERROR: 2022/04/17 21:55:38 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 21:55:44 [Categories for given ID doesn't exists - ]
ERROR: 2022/04/17 21:55:44 [0xc000004c90]
ERROR: 2022/04/17 21:57:34 [0xc00049b740]
ERROR: 2022/04/17 22:01:10 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 22:01:19 [0xc000004588]
ERROR: 2022/04/17 22:04:09 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 22:04:15 [Failed to unmarshal document fetched from DB - parsing time "2022-04-17 21:56:48.825351 +0530 IST m=+72.105112801" as "2006-01-02T15:04:05Z07:00": cannot parse " 21:56:48.825351 +0530 IST m=+72.105112801" as "T"]
ERROR: 2022/04/17 22:04:15 [0xc0004ba0c0]
ERROR: 2022/04/17 22:18:33 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 22:18:51 [Failed to unmarshal document fetched from DB - parsing time "2022-04-17 21:56:48.825351 +0530 IST m=+72.105112801" as "2006-01-02T15:04:05Z07:00": cannot parse " 21:56:48.825351 +0530 IST m=+72.105112801" as "T"]
ERROR: 2022/04/17 22:18:51 [0xc00042a630]
ERROR: 2022/04/17 22:20:22 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 22:21:04 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 22:21:18 [Failed to unmarshal document fetched from DB - parsing time "2022-04-17 22:21:13.9301872 +0530 IST m=+10.690152701" as "2006-01-02T15:04:05Z07:00": cannot parse " 22:21:13.9301872 +0530 IST m=+10.690152701" as "T"]
ERROR: 2022/04/17 22:21:18 [0xc000005458]
ERROR: 2022/04/17 22:22:31 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
ERROR: 2022/04/17 22:22:50 [Categories for given ID doesn't exists - ]
ERROR: 2022/04/17 22:22:50 [0xc000005188]
ERROR: 2022/04/17 22:23:49 [Failed to unmarshal document fetched from DB - parsing time "2022-04-17 22:23:17.2463019 +0530 IST m=+46.895195501" as "2006-01-02T15:04:05Z07:00": cannot parse " 22:23:17.2463019 +0530 IST m=+46.895195501" as "T"]
ERROR: 2022/04/17 22:23:49 [0xc0000052d8]
ERROR: 2022/04/17 22:24:38 [Failed to unmarshal document fetched from DB - parsing time "2022-04-17 22:24:35.5829491 +0530 IST m=+125.231842701" as "2006-01-02T15:04:05Z07:00": cannot parse " 22:24:35.5829491 +0530 IST m=+125.231842701" as "T"]
ERROR: 2022/04/17 22:24:38 [0xc0000048b8]
ERROR: 2022/04/17 22:34:26 [Connection to dynamoDB failed.]
ERROR: 2022/04/17 22:34:26 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: 0K3GLU3LRNKJNNTO0CUI0AIOAVVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/19 20:19:28 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 20:19:29 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: F74HLFFPBM88K8B6HDAMU2D5U7VV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/19 20:20:20 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 20:20:20 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: 06AB4CVIGI00AFDQGO0TU6FJRNVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/19 20:22:07 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 20:22:07 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: 1N1PFLJV519PU5SDBC5UF7331JVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/19 22:02:41 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 22:02:41 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: M80QN9NUILKMEF3BPPBK69H54BVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/19 22:02:58 [Database connection is down.]
ERROR: 2022/04/19 22:03:04 [Database connection is down.]
ERROR: 2022/04/19 22:05:09 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 22:05:09 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: J1DPG5SE95R6GIS74M9I6LIPQJVV4KQNSO5AEMVJF66Q9ASUAAJG]
INFO: 2022/04/19 22:06:22 [Created the table{
  TableDescription: {
    AttributeDefinitions: [{
        AttributeName: "id",
        AttributeType: "S"
      }],
    CreationDateTime: 2022-04-19 16:36:39.727 +0000 UTC,
    ItemCount: 0,
    KeySchema: [{
        AttributeName: "id",
        KeyType: "HASH"
      }],
    ProvisionedThroughput: {
      NumberOfDecreasesToday: 0,
      ReadCapacityUnits: 10,
      WriteCapacityUnits: 10
    },
    TableArn: "arn:aws:dynamodb:us-east-1:937145466380:table/ShippingAddress",
    TableId: "b791b55f-4a92-4a34-938b-f8809027673b",
    TableName: "ShippingAddress",
    TableSizeBytes: 0,
    TableStatus: "CREATING"
  }
}]
ERROR: 2022/04/19 22:06:50 [Got error calling CreateTable: %s ResourceInUseException: Table already exists: ShippingAddress]
INFO: 2022/04/19 22:07:18 [Created the table{
  TableDescription: {
    AttributeDefinitions: [{
        AttributeName: "id",
        AttributeType: "S"
      }],
    CreationDateTime: 2022-04-19 16:37:35.214 +0000 UTC,
    ItemCount: 0,
    KeySchema: [{
        AttributeName: "id",
        KeyType: "HASH"
      }],
    ProvisionedThroughput: {
      NumberOfDecreasesToday: 0,
      ReadCapacityUnits: 10,
      WriteCapacityUnits: 10
    },
    TableArn: "arn:aws:dynamodb:us-east-1:937145466380:table/ShippingAddress",
    TableId: "251b1055-005e-418f-abcc-4f182a052767",
    TableName: "ShippingAddress",
    TableSizeBytes: 0,
    TableStatus: "CREATING"
  }
}]
ERROR: 2022/04/19 22:32:53 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 22:32:54 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: Q5GN3TV12HUNUR84BLOKR8PTN7VV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/19 22:53:07 [Connection to dynamoDB failed.]
ERROR: 2022/04/19 22:53:07 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: 9I7846O7DCT2K68OP0RV3OESOVVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/20 11:35:15 [Connection to dynamoDB failed.]
ERROR: 2022/04/20 11:35:15 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: DG71RTJMUIETNF98APT453599BVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/20 16:52:55 [Connection to dynamoDB failed.]
ERROR: 2022/04/20 16:52:55 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: F96TOA396GIVTAVC8BJA8SRUS3VV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/20 16:54:54 [Connection to dynamoDB failed.]
ERROR: 2022/04/20 16:54:55 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: KKJUP2L2FV9GMAOTGVC6UHFMSRVV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/20 17:01:07 [Connection to dynamoDB failed.]
ERROR: 2022/04/20 17:01:07 [Got error calling CreateTable: %s UnrecognizedClientException: The security token included in the request is invalid.
	status code: 400, request id: RVO178G6F5HJFQ94BKVS774J67VV4KQNSO5AEMVJF66Q9ASUAAJG]
ERROR: 2022/04/20 17:01:26 [Database connection is down.]
